tfx/examples/cifar10/cifar10_utils_native_keras.py

```diff
@@ -0,0 +1,338 @@
+    std: The standard deviation used to normalize input image tensor
+  """
+
+  model_meta = _metadata_fb.ModelMetadataT()
+
+  # Creates input normalization info.
+  input_meta = _metadata_fb.TensorMetadataT()
+  input_normalization = _metadata_fb.ProcessUnitT()
+  input_normalization.optionsType = (
+      _metadata_fb.ProcessUnitOptions.NormalizationOptions)
+  input_normalization.options = _metadata_fb.NormalizationOptionsT()
+  input_normalization.options.mean = mean
+  input_normalization.options.std = std
+  input_meta.processUnits = [input_normalization]
+
+  # Creates output label map info.
+  output_meta = _metadata_fb.TensorMetadataT()
+  label_file = _metadata_fb.AssociatedFileT()
+  label_file.name = os.path.basename(label_map_path)
+  label_file.type = _metadata_fb.AssociatedFileType.TENSOR_AXIS_LABELS
+  output_meta.associatedFiles = [label_file]
+
+  # Creates subgraph info.
+  subgraph = _metadata_fb.SubGraphMetadataT()
+  subgraph.inputTensorMetadata = [input_meta]
+  subgraph.outputTensorMetadata = [output_meta]
+  model_meta.subgraphMetadata = [subgraph]
+
+  b = flatbuffers.Builder(0)
+  b.Finish(
+      model_meta.Pack(b),
+      _metadata.MetadataPopulator.METADATA_FILE_IDENTIFIER)
+  metadata_buf = b.Output()
+
+  # Populates metadata and label file to the model file.
+  populator = _metadata.MetadataPopulator.with_model_file(model_path)
+  populator.load_metadata_buffer(metadata_buf)
+  populator.load_associated_files([label_map_path])
+  populator.populate()
+
+
+# TFX Trainer will call this function.
+def run_fn(fn_args: TrainerFnArgs):
+  """Train the model based on given args.
+
+  Args:
+    fn_args: Holds args used to train the model as name/value pairs.
+  """
+  tf_transform_output = tft.TFTransformOutput(fn_args.transform_output)
+  train_dataset = _input_fn(fn_args.train_files, tf_transform_output, True,
+                            _TRAIN_BATCH_SIZE)
+  eval_dataset = _input_fn(fn_args.eval_files, tf_transform_output, False,
+                           _EVAL_BATCH_SIZE)
+
+  mirrored_strategy = tf.distribute.MirroredStrategy()
+  with mirrored_strategy.scope():
+    model, base_model = _build_keras_model()
+
+  # Our training regime has two phases: we first freeze the backbone and train
+  # the newly added classifier only, then unfreeze part of the backbone and
+  # fine-tune with classifier jointly.
+  steps_per_epoch = _TRAIN_DATA_SIZE / _TRAIN_BATCH_SIZE
+  total_epochs = int(fn_args.train_steps / steps_per_epoch)
+  classifier_epochs = int(total_epochs / 2)
+  finetune_epochs = total_epochs - classifier_epochs
+
+  absl.logging.info('Start training the top classifier')
+  # Freeze the whole MobileNet backbone and train the top classifer only
+  _freeze_model_by_percentage(base_model, 1.0)
+  # We need to recompile the model because layer properties have changed
+  model.compile(
+      loss='sparse_categorical_crossentropy',
+      optimizer=tf.keras.optimizers.RMSprop(lr=3e-4),
```

Comment:
Lets make the learning rates (and other constants in the file, top-level constants).